node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -398,6 +452,37 @@ func (o OsDeviceConnectivityHelperGeneric) GetHostsIdByArrayIdentifier(arrayIden
 
 }
 
+func (o OsDeviceConnectivityHelperGeneric) IsMpathHasRightVolumeId(dmPath string,
```

Comment:
```suggestion
func (o OsDeviceConnectivityHelperGeneric) IsMpathMatchVolumeId(dmPath string,
```
we generally use `Match` and not `Right`

Reply:
done